modules/update-preparing-unrecommended.adoc

```diff
@@ -0,0 +1,9 @@
+// Module included in the following assemblies:
+//
+// * updating/preparing_for_updates/updating-cluster-prepare.adoc
+
+:_content-type: CONTEXT
+[id="update-preparing-unrecommended"]
+= Assessing updates that are not recommended 
+
+Following recommended update guidelines ensures that the update has been tested and validated. Updating clusters by unrecommended paths introduces potential risk, and a successful update becomes less likely. 
```

Comment:
```suggestion
The following recommended update guidelines ensure that the update has been tested and validated. Updating clusters by unrecommended paths introduces potential risks, and a successful update becomes less likely. 
```

Reply:
Since the following guidelines are not recommended and we're referring to the alternative, I clarified by adding "the" after "following". I agree that sounded weird at first :)